source/PlayState.hx

```diff
@@ -168,13 +173,19 @@ class PlayState extends FlxState
 		FlxG.overlap(player, dtmEnemy1, dtmEnemy1.playerHitResolve);
 		FlxG.overlap(player, bullets, bulletHitPlayer);
 		FlxG.overlap(player, trap._grpBarTrap, trap.playerTrapResolve);
+		FlxG.overlap(player, sword, player.pickup_item);
+		FlxG.overlap(sword.hitbox.hitboxFrames, dtmEnemy1, sword.hit_enemy);
 		
 		// Add collision logic
 		FlxG.collide(blockGroup, player);
 		FlxG.collide(player, sentry1);
 		FlxG.collide(map, dtmEnemy1);
 		FlxG.collide(map, bullets);
 		FlxG.collide(blockGroup, bullets);
+		if(player.equipped_item != sword){
```

Comment:
This works for the moment, but it will have to be overhauled if more than one sword is in a level or other items are introduced.  It might be better to add all items to a group, run collide on that, and remove items from the group when they are picked up (making sure to add them back when dropped).